auth-jwt-service/src/test/java/io/stargate/auth/jwt/AuthzJwtServiceTest.java

```diff
@@ -98,7 +99,8 @@ public void executeDataReadWithAuthorization() throws Exception {
             "keyspace",
             "table",
             typedKeyValues,
-            SourceAPI.CQL);
+            SourceAPI.CQL,
+            EMPTY_HEADERS);
```

Comment:
Still going through but do we have tests anywhere that pass something other than an empty map?

Reply:
it is verified in C2 tests by passing tenant_id and verifying the full flow. I think the c2 is a better place for those tests because it actually uses the passed headers.